decompiler/util/DecompilerTypeSystem.cpp

```diff
@@ -422,7 +422,7 @@ int DecompilerTypeSystem::get_format_arg_count(const std::string& str) const {
 
   static const std::vector<char> single_char_ignore_list = {'%', 'T'};
   static const std::vector<std::string> multi_char_ignore_list = {
-      "0L", "1L", "3L", "1k", "1K", "2j", "0k", "0K", "30L", "1T", "2T"};
+      "0L", "1L", "3L", "1k", "1K", "2j", "0k", "30L", "1T", "2T", "100", "200", "350"};
```

Comment:
You removed `0K` from here.

Also what are the numbers for?

Reply:
Mis-identified strings for format strings I encountered -- instead of adding them to hacks I added them here.  But didn't add the entire token because the code i wrote here more than a year ago is dumb and only supports 2/3 char multi-char strings.

Reply:
you should probably add the specific strings to `bad_format_strings` instead then? so it's less confusing.

Reply:
I intend to this this part of code in the coming days / if not before this PR merges